<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="membershipMemberComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="membershipMemberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Jäsen</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="membershipJoinDateEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="membershipJoinLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Liittymis-päivä</string>
   </property>
  </widget>
  <widget class="QComboBox" name="membershipGroupComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="membershipGroupLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ryhmä</string>
   </property>
  </widget>
  <widget class="QLabel" name="membershipShareLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Osuus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="membershipCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="membershipAddPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisää</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="membershipShareSpinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="membershipGroupLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
  <widget class="QComboBox" name="membershipPartyComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>membershipMemberComboBox</tabstop>
  <tabstop>membershipGroupComboBox</tabstop>
  <tabstop>membershipShareSpinBox</tabstop>
  <tabstop>membershipJoinDateEdit</tabstop>
  <tabstop>membershipCancelPushButton</tabstop>
  <tabstop>membershipAddPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
